src/store.js

```diff
@@ -212,6 +217,32 @@ Store.prototype.setSelected = function(featureIds, options = {}) {
 };
 
 /**
+ * Sets the store's points selection, clearing any prior values.
+ * Expects to be passed the feature id that the points belong to.
+ * @param {string} featureId
+ * @param {Array<Array<string>>} points
+ * @return {Store} this
+ */
+Store.prototype.setSelectedPoints = function(featureId, points) {
+  this._selectedPointsFeatureId = featureId;
+  this._selectedPoints = points;
+  this._emitSelectionChange = true;
```

Comment:
Lets make sure to update what is getting emitted as well [here](https://github.com/mapbox/mapbox-gl-draw/blob/bbfd25cba32761207f01e8953f86891f076bcb69/src/render.js#L58)